policies/elections.tex

```diff
@@ -282,29 +295,29 @@ \subsubsection{Allegations, Decisions, Penalties, Appeals}
 		\hline 
 		Impersonation of another candidate or committee & 11 points \\
 		\hline
-		Campaigning outside of campaign period & 2 points, up to a maximum of 8 \\
+		Campaigning outside of campaign period & 2 to 8 points (depending on severity) \\
 		\hline
 		Speak to or email a class without the permission of the instructor & 2 points \\
 		\hline
-		Directly providing a method of voting to voters during polling period, or campaigning to a voter while voting & 3 points \\
+		Directly providing a method of voting to voters during polling period (this does not include telling voters that they can vote at \votewebsite, but does include handing them an electronic device with \votewebsite{} opened, for example), or campaigning to a voter while voting & 3 points \\
 		\hline
-		Distribute money, drugs, alcohol, or any other form of bribery, for campaign purposes & 11 points \\
-		\hline 
-		Enter into contractual obligations with any group(s) on or off campus that may affect MathSoc during their term in office & 11 points \\ 
+		Distributing bribes for campaign purposes & 7 to 11 points (depending on severity) \\
```

Comment:
Distributing bribes containing alcohol, or any illicit items, for campaign purposes during campaigning period & 7 to 11 points (depending on severity); Distributing bribes for campaign purposes during voting period period & 11 points

Reply:
Above is wrong, what we passed in the meeting was:

“Distributing bribes, including food, for campaigning purposes during campaigning period & 7 to 11 points (depending on severity); Distributing bribes for campaign purposes during voting period period & 11 points”